lib/hca/enrollment_system.rb

```diff
@@ -644,16 +644,25 @@ def convert_hash_values!(hash)
       hash.delete_if { |_k, v| v.blank? }
     end
 
-    def build_form_for_user(current_user)
+    def get_user_variables(user_identifier)
+      return [nil, nil] if user_identifier.blank?
```

Comment:
This line is functionally redundant with line 658.  That is, with this line here, the `else` path can never occur.
I'd probably remove this line, so that the `if` maintains an `else` path and clearly lays out the logic.
What do you think?

Reply:
the else path can occur if `user_identifier` is `{ icn: nil, edipi: nil }` which would happen if the user is loa1

Reply:
Yup. My mistake.  For some reason I was thinking a hash with all `nil` values would respond to `blank?` with `true`